app/components/search/Refinements/BrowseRefinementList.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { InstantSearch } from "react-instantsearch-core";
+import { render, screen, waitFor } from "@testing-library/react";
+import BrowseRefinementList from "components/search/Refinements/BrowseRefinementList";
+import { createSearchClient } from "../../../../test/helpers/createSearchClient";
+
+describe("BrowseRefinementList", () => {
+  test("renders the correct the number of categories", async () => {
```

Comment:
**question (non-blocking):** There are 12 categories in searchClient but expected is 10. Is there anything in the repo that explains this?

Reply:
Good question. I updated the test name to indicate more explicitly what this is testing.